docs/packages.rst

```diff
@@ -18,10 +18,11 @@ Packages
   docs_inspector/sdk_index
   docs_neal/sdk_index
   docs_dnx/sdk_index
+  docs_preprocessing/sdk_index
+  docs_samplers/index
```

Comment:
Consider an sdk_index (even though we're not building for the repo, be good to keep consistent)

Reply:
I think we should be updating the other ones to drop `sdk_index` and just use `index`